### ISSUE TITLE
Does Location need a type field?

### ISSUE THREAD

**Issue by pmackay** [folklabs/openlocal#8]
Would it be good to have a type field that could define select from a list like these below?

Gallery or Craft centre
Playschool
School
Market
Restaurant, Pub or Bar
Council office
Sports centre
Community centre
Shop
Meeting room?
Park or garden
Place of worship
Station

@djwesto @GilesGibson

**Comment by djwesto** [comment]
Good idea in principle but I'm anxious about creating a new taxonomy. Is there any chance we could use an existing taxonomy we already have within OL?

And is this based on Open Streetmap's taxonomy?
